### PR TITLE
INTYGFV-14567: Upload encrypted package to Sjut.

### PR DESCRIPTION
**Pull request by mww-sopra-steria**

### CODE REVIEW

File: customer-terminate-service/app/build.gradle [hunk 1]
```diff
@@ -33,3 +33,7 @@ tasks.named('test') {
     useJUnitPlatform {
     }
 }
+
+bootRun {
+    systemProperty("spring.profiles.active", "testability,dev")
```
[the remark this answers is not in the document]
Comment (mww-sopra-steria) [reply customer-terminate-service/app/build.gradle:38]: Update README.

**Review by IamFriberg — DISMISSED** [review]
Nice!

File: customer-terminate-service/app/src/main/java/se/inera/intyg/cts/infrastructure/integration/sjut/UploadPackageToSjut.java [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+  @Override
+  public void uploadPackage(Termination termination, File packageToUpload) {
+    final var resource = getResource(packageToUpload);
+    final var packageMetadata = getPackageMetadata(termination);
+
+    final var multipartBodyBuilder = new MultipartBodyBuilder();
+    multipartBodyBuilder.part(FILE_PART, resource);
+    multipartBodyBuilder.part(METADATA_PART, packageMetadata);
+
+    final var clientReponse = webClient.post()
```
[the remark this answers is not in the document]
Comment (mww-sopra-steria) [reply customer-terminate-service/app/src/main/java/se/inera/intyg/cts/infrastructure/integration/sjut/UploadPackageToSjut.java:65]: Introduced unit tests for UploadPackageToSjut based on your suggestion with the mockServer.

File: customer-terminate-service/app/src/test/java/se/inera/intyg/cts/infrastructure/integration/UploadPackageToMemory.java [hunk 3]
```diff
@@ -0,0 +1,13 @@
+package se.inera.intyg.cts.infrastructure.integration;
+
+import java.io.File;
+import se.inera.intyg.cts.domain.model.Termination;
+import se.inera.intyg.cts.domain.service.UploadPackage;
+
+public class UploadPackageToMemory implements UploadPackage {
+
+  @Override
+  public void uploadPackage(Termination termination, File packageToUpload) {
```
[the remark this answers is not in the document]
Comment (mww-sopra-steria) [reply customer-terminate-service/app/src/test/java/se/inera/intyg/cts/infrastructure/integration/UploadPackageToMemory.java:10]: :-)

File: customer-terminate-service/domain/src/test/java/se/inera/intyg/cts/domain/service/UploadPackageToMemory.java [hunk 4]
```diff
@@ -0,0 +1,24 @@
+package se.inera.intyg.cts.domain.service;
+
+import java.io.File;
+import se.inera.intyg.cts.domain.model.Termination;
+
+public class UploadPackageToMemory implements UploadPackage {
+
```
[the remark this answers is not in the document]
Comment (mww-sopra-steria) [reply customer-terminate-service/domain/src/test/java/se/inera/intyg/cts/domain/service/UploadPackageToMemory.java:7]: :-)

**Review by IamFriberg — APPROVED** [review]
Nice!

**Pull request merged**